mail/application_test.go

```diff
@@ -109,7 +112,8 @@ func (s *ApplicationTestSuite) TestQueueMail() {
 		Cc([]string{testCc}).
 		Bcc([]string{testBcc}).
 		Attach([]string{"../logo.png"}).
-		Content(mail.Content{Subject: "Goravel Test Queue", Html: "<h1>Hello Goravel</h1>"}).
+		Subject("Goravel Test Queue").
+		Content(mail.Content{Html: "<h1>Hello Goravel</h1>"}).
 		Queue())
```

Comment:
Missing test cases for `Queue` with variable.

Reply:
Yeah test cases are no finished yet.